docs/en/05_Contributing/04_Making_A_SilverStripe_Core_Release.md

```diff
@@ -148,8 +152,15 @@ If you are managing a release, it's best to first make sure that SilverStripe ma
 are aware of any impending release. This is so that they can ensure that a relevant blog
 post will appear on [www.silverstripe.org/blog](http://www.silverstripe.org/blog/), and
 cross-posted to other relevant channels such as Twitter and Facebook.
-Make sure Nicole and Vinh at silverstripe at aware by sending an email to
-[marketing@silverstripe.com](mailto:marketing@silverstripe.com).
+Sending an email to [marketing@silverstripe.com](mailto:marketing@silverstripe.com)
+with an overview of the release and a rough release timeline.
+
+Check all tickets ([framework](https://github.com/silverstripe/silverstripe-framework/milestones), 
+[cms](https://github.com/silverstripe/silverstripe-cms/milestones), 
+[installer](https://github.com/silverstripe/silverstripe-installer/milestones)) assigned to that milestone are 
+either closed or reassigned to another milestone.
+
+Merge up from other older supported release branches (e.g. merge 3.1 into 3.2, then 3.2 into 3.3, then 3.3 into master).
```

Comment:
True, I've committed this straight to core throughhttps://github.com/silverstripe/silverstripe-framework/commit/70897834c98beccfaab5ad85ba009612dd024efc